vercel.json

```diff
@@ -36,6 +36,55 @@
 					"value": "public, max-age=31536000, immutable"
 				}
 			]
+		},
+		{
+			"source": "/(.*)",
+			"headers": [
+				{
+					"key": "Access-Control-Allow-Origin",
+					"value": "https://unicorn-utterances.com"
+				},
+				{
+					"key": "Content-Security-Policy",
+					"value": "default-src 'self';base-uri 'self';form-action 'self' https://unicorn-utterances.com/ https://app.convertkit.com/;frame-ancestors 'none';frame-src https:;img-src 'self' https: data:;object-src 'none';script-src 'self' 'unsafe-inline' https://vercel.live/;connect-src 'self' https://vercel.live/;style-src 'self' 'unsafe-inline';upgrade-insecure-requests"
+				},
+				{
+					"key": "Cross-Origin-Opener-Policy",
+					"value": "same-origin"
+				},
+				{
+					"key": "Cross-Origin-Resource-Policy",
+					"value": "same-origin"
+				},
+				{
+					"key": "Permissions-Policy",
+					"value": "camera=(), display-capture=(), document-domain=(), geolocation=(), gyroscope=(), magnetometer=(), microphone=(), midi=(), payment=(), usb=(), clipboard-read=(), clipboard-write=(self)"
+				},
+				{
+					"key": "Referrer-Policy",
+					"value": "no-referrer"
+				},
+				{
+					"key": "X-Content-Type-Options",
+					"value": "nosniff"
+				},
+				{
+					"key": "X-Download-Options",
+					"value": "noopen"
+				},
+				{
+					"key": "X-Frame-Options",
+					"value": "DENY"
+				},
+				{
+					"key": "X-Permitted-Cross-Domain-Policies",
+					"value": "DENY"
+				},
+				{
+					"key": "X-XSS-Protection",
+					"value": "0"
```

Comment:
Curious to know why we'd disable XSS protection

Reply:
[According to MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection): "Even though this feature can protect users of older web browsers that don't yet support CSP, in some cases, XSS protection can create XSS vulnerabilities in otherwise safe websites." There are a variety of recommendations to use `X-XSS-Protection: 0` because of this - which is also in [helmet.js](https://github.com/helmetjs/helmet#x-xss-protection) and most major sites.